images/rabbitmq/Dockerfile

```diff
@@ -3,9 +3,26 @@ FROM rabbitmq:3-management
 ARG LAGOON_VERSION
 ENV LAGOON_VERSION=$LAGOON_VERSION
 
-COPY rabbitmq_delayed_message_exchange-3.7.0.ez /plugins
+ENV RABBITMQ_DEFAULT_USER=guest \
+    RABBITMQ_DEFAULT_PASS="guest"\
+    RABBITMQ_DEFAULT_HA_PATTERN='^$'\
+    RABBITMQ_DEFAULT_VHOST='/'
 
+RUN apt-get -y update \
+    && apt-get -y install curl \
+    && curl -sLo /bin/ep https://github.com/kreuzwerker/envplate/releases/download/1.0.0-RC1/ep-linux \
```

Comment:
as we already have ep in our commons image we could easily take it from there that would make the code a bit cleaner

see - https://github.com/amazeeio/lagoon/blob/master/images/docker-host/Dockerfile#L13

Reply:
Good to know! Changed to COPY from commons